docs/_publish/launch-process/user-help.md

```diff
@@ -1,20 +1,86 @@
 ---
-title: Create help documentation for your users
+title: Best practices for creating exceptional documentation
 order: 4
 layout: post-toc
 redirect_from: 
 ---
 
 # Create help documentation for your users
 
+## Zapier-hosted help documentation
+
 The Zapier team provide frontline support for your integration, and in order to provide the best experience for your users, help documentation about using your integration in Zaps is hosted in the [Zapier Help Center](https://help.zapier.com/hc/en-us). 
 
-Ensure the Zapier Technical Writing team can create accurate documentation by filling out [this form](https://eu.jotform.com/form/202233475923352). 
+Ensure the Zapier Technical Writing team can create accurate documentation by filling out [this form](https://eu.jotform.com/form/202233475923352). Once submitted, it takes roughly 2 weeks to get help docs published in the Help Center. Optionally include links out to documentation on your app's site to help customers find the information they are looking for.
+
+As you maintain and add functionality to your integration, use the same form to submit updates at any time to the Technical Writing team. When adding Zapier logos on any help documentation or pages in your app, access the up-to-date branding assets [here](https://brand.zapier.com/).
+
+## Self-hosted help documentation
+
+Step-up your documentation game. Implementing these essential strategies and tips can ensure that your audience gets the assistance they need, seamlessly. Elevate your documentation game today and revolutionize your approach to creating help docs!
```

Comment:
I would suggest removing this line, but that's just my opinion 😊

> Elevate your documentation game today and revolutionize your approach to creating help docs!

During the rewrite of these docs, we removed a lot of phrases or sentences that ended in !!, when they did not add to the informative content these docs should provide.